### PR TITLE
FE: Profile listings UI

### PR DESCRIPTION
**Pull request by Habib-Ahmad**
### What this PR does (required):
- Displays a list of profiles
- Each profile is a UserCard component which is mapped in ProfileListings component
- The profileListings component is displayed on the dashboard

### Screenshots / Videos (front-end only):
image

### Any information needed to test this feature (required):
- User needs to login and navigate to the dashboard

### CODE REVIEW

File: client/src/components/ProfileListings/ProfileListings.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import { Grid } from '@mui/material';
+import UserCard from '../UserCard/UserCard';
+
+const ProfileListings = () => {
+  return (
+    <Grid container rowSpacing={4} sx={{ padding: '0 5vw' }}>
+      {dummyData.map((user) => (
+        <Grid item display="flex" justifyContent="center" xs={12} sm={6} md={4} key={user.name}>
```
Comment (moffatethan) [inline client/src/components/ProfileListings/ProfileListings.tsx:8]: Make sure to add. `key` prop onto the Grid item.

**Review by moffatethan — CHANGES_REQUESTED** [review]
Just one comment

File: client/src/components/ProfileListings/ProfileListings.tsx (hunk 1 above)
Comment (Habib-Ahmad) [reply client/src/components/ProfileListings/ProfileListings.tsx:8]: There is a key prop at the end of line 8

Comment (moffatethan) [reply client/src/components/ProfileListings/ProfileListings.tsx:8]: Oh yes - didn't see it there!

**Review by moffatethan — APPROVED** [review]
Good to go

**Pull request merged**